packages/pangea-node-sdk/.gitlab-ci.yml

```diff
@@ -78,29 +127,124 @@ generate_docs:
     paths: ["packages/pangea-node-sdk/docs.json"]
     when: on_success
 
-test_pangea_node_sdk:
-  extends: .pangea_node_sdk_base
-  stage: Unit Test
+pangea-node-sdk-unit-tests:
+  extends: .pangea-node-sdk-base
+  stage: unit_tests
   script:
     - yarn test:unit
 
-integration_pangea_node_sdk:
-  extends: .pangea_node_sdk_base
-  stage: Integration Test
+pangea-node-sdk-it-audit:
+  extends: .pangea-node-sdk-test-base
+  variables:
+    ENV: ${SERVICE_AUDIT_ENV}
+  script:
+    - npx cross-env NODE_OPTIONS="--experimental-vm-modules --openssl-legacy-provider" npx jest --testPathPattern=tests/integration/audit.test.ts
```

Comment:
I think we can move `NODE_OPTIONS` to the `variables` section of `.pangea-node-sdk-test-base`.

```yaml
variables:
  NODE_OPTIONS: "--experimental-vm-modules --openssl-legacy-provider"
```